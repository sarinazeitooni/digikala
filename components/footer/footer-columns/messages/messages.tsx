import InstagramIcon from "@material-ui/icons/Instagram";
import TwitterIcon from "@material-ui/icons/Twitter";
import LinkedInIcon from "@material-ui/icons/LinkedIn";
import GroupWorkIcon from "@material-ui/icons/GroupWork";
import React from "react";

const messages = {
    links:[
        <InstagramIcon/>,
        <TwitterIcon/>,
        <LinkedInIcon/>,
        <GroupWorkIcon/>
    ],
    columns : [
        {
            social: false,
            title: 'با دیجی کالا',
            items : [
                'اتاق خبر دیجی کلالا',
                'فروش در دیجی کالا',
                'فرصت های شغلی',
                'تماس با دیجی کالا',
                'درباره دیجی کالا'
            ]
        },
        {
            social: false,
            title: 'خدمات مشتریان',
            items : [
                'پاسخ به پرسش های متداول',
                'رویه های بازگرداندن کالا',
                'شرایط استفاده',
                'حریم خصوصی',
                'گزارش باگ'
            ]
        },
        {
            social: false,
            title: 'راهنمای خرید از دیجی کالا',
            items : [
                'نحوه ثبت سفارش',
                'رویه ارسال سفارش',
                'شیوه های پرداخت'
            ]
        },
        {
            social : true,
            title: 'با ما همراه باشید',
            links : 'با ما همراه باشید',
            news : 'از جدیدترین تخفیف ها باخبر شوید',
            email : 'آدرس ایمیل خود را وارد کنید',
            submit : 'ثبت'
        }
    ]
};
export default messages;