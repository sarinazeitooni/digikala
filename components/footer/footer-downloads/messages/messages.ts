const messages = {
    downloadTitle : 'دانلود اپلیکیشن دیجی کالا',
    urls : [
        'https://www.digikala.com/static/files/6f0c9aeb.svg',
        'https://www.digikala.com/static/files/b43d144f.svg',
        'https://www.digikala.com/static/files/401848fb.png',
        'https://www.digikala.com/static/files/c824b056.svg'
    ]
};
export default messages;