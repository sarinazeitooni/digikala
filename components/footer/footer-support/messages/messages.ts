const messages = {
    backToTop: 'بازگشت به بالا',
    supportFirst: 'تلفن پشتیبانی: ۶۱۹۳۰۰۰۰ - ۰۲۱',
    supportSecond : 'هفت روز هفته، ۲۴ ساعت شبانه‌روز پاسخگوی شما هستیم.',
    features: [
        {
            icon: 'https://www.digikala.com/static/files/8f570b58.svg',
            text: 'امکان تحویل اکسپرس'
        },
        {
            icon: 'https://www.digikala.com/static/files/22414818.svg',
            text: 'مکان پرداخت در محل'
        },
        {
            icon: 'https://www.digikala.com/static/files/a9286d2f.svg',
            text: '7 روز هفتهُ 24 ساعته'
        },
        {
            icon: 'https://www.digikala.com/static/files/514926b1.svg',
            text: '7روز ضمانت بازگشت کالا'
        },
        {
            icon: 'https://www.digikala.com/static/files/fdb293e6.svg',
            text: 'ضمانت اصل بودن کالا'
        }
    ]
};
export default messages;