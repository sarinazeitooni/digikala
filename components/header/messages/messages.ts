const messages = {
    inputPlaceHolder : "جستجو در دیجی‌کالا …",
    signIn : "ورود به حساب کاربری",
    location : 'لطفا شهر و استان خود را انتخاب کنید',
    navigation : [
        {
            title : 'دسته بندی کالاها'
        },
        {
            title : 'سوپرمارکت'
        },
        {
            title : 'تخفیف ها و پیشنهادها'
        },
        {
            title : 'دیجی کالای من'
        },
        {
            title : 'دیجی پلاس'
        },
        {
            title : 'دیجی کلاب'
        },
        {
            title : 'دیجی پی'
        },
        {
            title : 'سوالی دارید؟'
        },
        {
            title : 'فروشنده شوید'
        }
    ]
};
export default messages;