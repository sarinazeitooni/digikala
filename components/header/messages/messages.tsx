import MenuIcon from '@material-ui/icons/Menu';
import StorefrontIcon from '@material-ui/icons/Storefront';
import MoneyOffIcon from '@material-ui/icons/MoneyOff';
import CheckCircleOutlineIcon from '@material-ui/icons/CheckCircleOutline';
import CategoryIcon from '@material-ui/icons/Category';
import PaymentIcon from '@material-ui/icons/Payment';
const messages = {
    inputPlaceHolder : "     جستجو در دیجی‌کالا …",
    signIn : "ورود به حساب کاربری",
    logIn : 'ورود',
    location : 'لطفا شهر و استان خود را انتخاب کنید',
    navigation : [
        {
            icon : <MenuIcon/>,
            title : 'دسته بندی کالاها',
            categories : true ,
            discount : false ,
            digi : false

        },
        {   icon : <StorefrontIcon/>,
            title : 'سوپرمارکت',
            categories : false ,
            discount : false ,
            digi : false
        },
        {   icon : <MoneyOffIcon/>,
            title : 'تخفیف ها و پیشنهادها',
            categories : false ,
            discount : true ,
            digi : false
        },
        {   icon : <CheckCircleOutlineIcon/>,
            title : 'دیجی کالای من',
            categories : false ,
            discount : true ,
            digi : false
        },
        {   icon : <CategoryIcon/>,
            title : 'دیجی پلاس',
            categories : false ,
            discount : false ,
            digi : true,
            digiComponent : {
                url : 'https://www.digikala.com/static/files/38492329.jpg',
                menu : false,
                titleImage : 'https://www.digikala.com/static/files/4a2895fc.svg',
                width : '648px'
            }
        },
        {   icon : '',
            title : 'دیجی کلاب',
            categories : false ,
            discount : false ,
            digi : true,
            digiComponent : {
                url : 'https://dkstatics-public.digikala.com/digiclub/53aa31f81138fd588df842086e152102979a00d7_1601458094.jpg',
                titleImage : 'https://www.digikala.com/static/files/1c93eb76.svg',
                width : '718px',
                menu : true,
                menuItem : [
                    'صفحه اصلی دیجی کلاب'
                ]
            }
        },
        {   icon : PaymentIcon,
            title : 'دیجی پی',
            categories : false ,
            discount : false ,
            digi : true,
            digiComponent : {
                url : 'https://www.digikala.com/static/files/148f56c6.png',
                menu : true,
                width : '586px',
                menuItem : [
                    'فعالسازی کیف پول',
                    'صفحه اصلی دیجی پی',
                    'خرید شارژ',
                    'خرید بسته اینترنت',
                    'پرداخت قبض'
                ]
            }
        },
        {
            title : 'سوالی دارید؟',
            categories : false ,
            discount : false ,
            digi : false
        },
        {
            title : 'فروشنده  شوید',
            categories : false ,
            discount : false ,
            digi : false
        }
    ]
};
export default messages;