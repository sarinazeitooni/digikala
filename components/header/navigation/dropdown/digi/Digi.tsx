import style from './style/digi.module.scss';
import React from 'react';
import ArrowBackIosIcon from '@material-ui/icons/ArrowBackIos';
interface IProps {
    data: {url : string , titleImage : string , menu : boolean , width : string , menuItem : []}
}
const Digi :React.FC<IProps> = ({data})=>{
    return(
        <div style={{width : data.width}} className={style['Digi-container']}>
            <div className={style['image-container']}>
                <img src={data.url}/>
            </div>
            <div className={style['info-container']}>
                {data.titleImage && <img className={style['title-image']} src={data.titleImage}/>}
                {data.menu && data.menuItem.map((item)=>{
                    return(
                        <div key={item} className={style['menu-item']}>{item}<ArrowBackIosIcon/></div>
                    )
                })}
            </div>
        </div>
    )
}
export default Digi;