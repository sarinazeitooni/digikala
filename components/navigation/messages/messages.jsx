const messages = {
    placeholder: '          ... جستوجو در دیجی کالا       ',
    enter : 'ورود به حساب کاربری'
};
export default messages;