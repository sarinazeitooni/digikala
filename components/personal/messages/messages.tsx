const messages = {
    title : "شخصی سازی پیشنهادها!",
    subtitle : 'برای مشاهده پیشنهادهای مناسب خود و همچنین داشتن تجربه بهتر لطفا وارد حساب کاربری شوید.',
    login : 'ورود به حساب دیجی کالا',
    signup : 'ثبت نام رایگان درصورت نداشتن حساب کاربری'
};
export default messages;