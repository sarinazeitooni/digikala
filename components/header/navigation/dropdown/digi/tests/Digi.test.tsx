import React from 'react'
import {shallow} from 'enzyme';
import Digi from "../Digi";
import messages from "../../../../messages/messages";
describe('Digi component', () => {
    it('should render without throwing an error', ()=> {
        const wrapper = shallow(<Digi data={messages.navigation.map((item)=>{item.digiComponent})}/>);
        expect(wrapper).toMatchSnapshot();
    })
});