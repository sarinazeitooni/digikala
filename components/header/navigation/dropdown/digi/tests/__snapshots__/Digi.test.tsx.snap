// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Digi component should render without throwing an error 1`] = `ShallowWrapper {}`;
